plugins/dashboard-object-viewer/webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const { resolve } = require('path');
+const webpack = require('webpack');
```

Comment:
Could use vite library mode I think. But also this webpack config is pretty standard